### PR TITLE
31-3-22

### PR DESCRIPTION
**Pull request by umaaug**

### CODE REVIEW

File: kANe/v5p1/1_Vratas_and_Utsavas/03_vratAdhikArin_and_related_topics.md [hunk 1]
```diff
@@ -71,83 +71,81 @@ Purpose of most vratas
 
 55
 
-Another remarkable matter is that the Mahābhārata provides that brāhmaṇas and ksatriyas should not engage in a continuous fast for more than three days and that vaiśyas and $ūdras can observe a continuous fast for two days only and a fast for three days is not laid down for these two classes by those who know dharma. Devala also has a similar provision, 140
+Another remarkable matter is that the Mahābhārata provides that brāhmaṇas and kśatriyas should not engage in a continuous fast for more than three days and that vaiśyas and Śūdras can observe a continuous fast for two days only and a fast for three days is not laid down for these two classes by those who know _dharma_. Devala also has a similar provision[^140].
```
Comment (vvasuki) [inline kANe/v5p1/1_Vratas_and_Utsavas/03_vratAdhikArin_and_related_topics.md:74]: kṣatriyas

Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/03_vratAdhikArin_and_related_topics.md:74]: In IAST mode, for letter 's',  I get only 'ś' and not 'ṣ'. 
Please tell me how to get 'ṣ'

Comment (vvasuki) [reply kANe/v5p1/1_Vratas_and_Utsavas/03_vratAdhikArin_and_related_topics.md:74]: Try S or shh

Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/03_vratAdhikArin_and_related_topics.md:74]: I am not getting it with any combination.

Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/03_vratAdhikArin_and_related_topics.md]: corrected

**Pull request merged**